docs/conditional.rst

```diff
@@ -142,7 +122,7 @@ You can check for conditional operation failures by inspecting the cause of the
 .. code-block:: python
 
     try:
-        thread_item.save(forum_name__null=False)
+        thread_item.save(Thread.forum_name.exists())
     except PutError as e:
```

Comment:
Will try to put something together. I agree this isn't great, see:
https://github.com/pynamodb/PynamoDB/pull/108
https://github.com/pynamodb/PynamoDB/pull/112